src/styles/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  background: $white;
+  border: 1px solid $gray-elements;
+  border-radius: 5px;
+  max-width: 200px;
+  &__image {
+    width: 100%;
+    height: 134px;
+    background-image: url(../images/imac.jpeg);
+    background-position: center;
+    background-size: cover;
+    margin: 0 auto 40px;
+  }
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-accent;
+    margin-bottom: 4px;
+  }
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 1.4;
+    color: $secondary;
+  }
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    align-items: center;
+    margin-top: 16px;
+    margin-bottom: 24px;
+    &-reviews {
+      font-size: 10px;
+      line-height: 14px;
+      color: $main-accent;
```

Comment:
The variable `$main-accent` is used here. Ensure that it is defined in your SCSS files or imported from another file.